railties/lib/rails/paths.rb

```diff
@@ -175,6 +175,12 @@ def to_ary
         @paths
       end
 
+      def expanded_paths
```

Comment:
It feels weird to have both an `expanded_paths` and an `expanded` method - why can't we use `expanded`?